lib/download/download_preprocessor.rb

```diff
@@ -63,14 +63,14 @@ def DownloadPreprocessor.match_trim_elipsis(value, header)
   def DownloadPreprocessor.find_header(options)
     raise 'Request to find header requires a header name' if options[:header_name].nil? 
     raise 'Request to find header requires a handle' if options[:handle].nil?
-    header_in = options[:header_in].nil? ? 'col' : options[:header_in]
+    header_in = options[:header_in] || 'col'
     match_type = options[:match_type].nil? ? :hiwi : options[:match_type].parameterize.underscore.to_sym
-    search_main = options[:search_main].nil? ? 1 : options[:search_main]
-    cached_files = options[:cached_files].nil? ? DownloadsCache : options[:cached_files]
-    
+    search_main = options[:search_main] || 1
+    cached_files = options[:cached_files] || DownloadsCache.new
+
     spreadsheet = options[:sheet].nil? ? cached_files.csv(options[:handle]) : cached_files.xls(options[:handle], options[:sheet])
     
-    search_start = options[:search_start].nil? ? 1 : options[:search_start]
+    search_start = options[:search_start] || 1
```

Comment:
Luckily, we don't have any cases in the db of :search_start being 0. It looks like this parameter is 1-indexed. If it were 0-indexed this would have an edge case where we get the wrong :search_start value.